docs/reference/filters.md

```diff
@@ -465,6 +466,26 @@ Example:
 * -> repeatContent("I will not waste chalk. ", 1000) -> <shunt>;
 ```
 
+### wrapContent
+
+Add prefix and suffix to the response.
+
+Parameters:
+
+* suffix (string)
+* prefix (string)
+
+Examples:
+
+```
+* -> wrapContent("foo", "baz") -> inlineContent("bar") -> <shunt>
+```
+
+```
+// JSON array of 300 zeros
+* -> wrapContent("[", "0]") -> repeatContent("0, ", 297) -> <shunt>
```

Comment:
These are 298 "0", no?

Reply:
Yeah, the comment is wrong, should be `// JSON array of 100 zeros`

297 is the total length of repeated content, i.e 99 times the `"0, "`.

Reply:
Fixed the comment